plugins/core/jetbrains-community/tst/software/aws/toolkits/jetbrains/core/credentials/sso/SsoAccessTokenProviderTest.kt

```diff
@@ -300,6 +302,35 @@ class SsoAccessTokenProviderTest {
         verify(ssoCache).saveAccessToken(argThat<DeviceGrantAccessTokenCacheKey> { startUrl == ssoUrl }, eq(refreshedToken))
     }
 
+    @Test
+    fun `refresh access token error handling does not fail if AWS error details are missing`() {
+        val expirationClientRegistration = clock.instant().plusSeconds(120)
+        setupCacheStub(expirationClientRegistration)
+
+        val accessToken = DeviceAuthorizationGrantToken(ssoUrl, ssoRegion, "dummyToken", "refreshToken", clock.instant())
+        ssoCache.stub {
+            on(
+                ssoCache.loadAccessToken(argThat<DeviceGrantAccessTokenCacheKey> { startUrl == ssoUrl })
+            ).thenReturn(
+                accessToken
+            )
+        }
+
+        ssoOidcClient.stub {
+            on(
+                ssoOidcClient.createToken(refreshTokenRequest())
+            )
+                .thenThrow(
+                    AwsServiceException.builder()
+                        .awsErrorDetails(AwsErrorDetails.builder().errorMessage("Test Err Msg").build())
```

Comment:
shouldn't this be null if we're testing `if AWS error details are missing`?

Reply:
Yes. Fixing